docs/alincome/itemizedjobs.md

```diff
@@ -5,16 +5,26 @@ title: |
 sidebar_label: |
   Itemized Jobs
 slug: /alincome/itemizedjobs
+# Note(brycew): this has duplicated content with jobs.md.
+# Not sure the best way to avoid it, as the classes are
+# different enough for different pages IMO.
 ---
 
 Sometimes, you'll need detailed information about a person's
 income at the same job, including how much income comes from tips, or
 how much is deducted from their paycheck for different reasons. `ALItemizedJob` is custom built for this purpose.
 
-## Guiding Philosophy
-
 ## ALItemizedJob
 
+Sometimes, courts will ask for specific pieces of information about a person's pay, such as how many tips they received, or how much is deducted from their pay for health insurance. To keep track of all of those pieces of information for several jobs, you, the interview author,
```

Comment:
Caroline has said that the main reason she wants this is because otherwise the user doesn't keep track of everything they need to take into account and they have to do a lot of the math themselves. You say a bit about that at the bottom of this section, but maybe it's worth thinking about which of those is the main purpose. For example, I think I understood that Caroline preferred it as a user experience over ALJob.

Reply:
Hm, if that's the case, there isn't a strong reason for ALJob at all? Should we still even document it / suggest it, or should we steer everyone towards ItemizedJob? 

I see what she means, but TBH I'm not sure if ALItemizedJob is there yet in terms of UX. I know that's the purpose of a lot of our other classes, like ALExpenseList, where we can ask the user about a lot of different expenses so they don't have to keep track of them. But ItemizedJob still feels cumbersome when doing that, and it doesn't quite remove the need for users to make calculations (a lot more discussion of that in https://github.com/SuffolkLITLab/docassemble-ALToolbox/issues/140).

Reply:
Now that I've thought about https://github.com/SuffolkLITLab/docassemble-ALToolbox/issues/140, I'm realizing that the core functionality of ALItemizedJob of handling additional income like tips, commissions, and overtime wages isn't flexible enough; all of those additional incomes are inconsistent, and asking about them in aggregate either require the user to do math (i.e. directly asking "what's your usual amount" or "what's the average"), or they just get over or under estimated.

I did a quick "look at this website I made" user test with my partner, and she pointed out some of these inconsistencies and unclear instructions. I'll make an issue for it separately, but I do think that the current language should probably stay for now, and maybe I'll add a warning with the above info as well.

Also will directly invite @CaroRob to weigh in if she has any thoughts.